static/include/components/base-control.md

```diff
@@ -0,0 +1,5 @@
+After you configure the base, go to the [**Control**](/manage/fleet/robots/#control) tab and expand the base component to view the controls to enable keyboard or [discrete](https://docs.viam.com/try-viam/try-viam-tutorial/#discrete-movement-control) control over your robot's movement.
+If your base makes use of a [camera](/components/camera/) component, the **Control** tab for your base will also display a **Live Feed** section.
+This section contains buttons to toggle cameras on or off and to arrange how you view the camera feeds.
+
+![The base component in control tab](/components/base/base-control-tab.png)
```

Comment:
This screenshot as-is appears quite large when [rendered on-page](https://docs-test.viam.dev/036c15e88047529af7ef8c5b3416facd5eefd05b/public/components/base/fake/#test-the-base). We have another way of including images that can resize the image during the build, that looks like this:

```
{{< imgproc src="/components/base/base-control-tab.png" alt="The base component in control tab" resize="300x" >}}
```

If you stage your PR locally with `hugo serve-dev` or similar, you can see how it looks and then tweak that size (`400`? `250` ?) until it looks just right by you. Image sizing does not have to be perfect: try to aim for much of the text in the image to be about the same size as the surrounding copy!